README.md

```diff
@@ -46,6 +46,18 @@ curl http://localhost:8000/api/tables
 
 It should return a list of the tables inside `sqlite.db` database.
 
+**Updating a user**
```

Comment:
@AbegaM - the CLI is just for editing stuff related to super-users (setting flag or changing pwd for superuser account).  The CLI is also used to remove superuser status, so that's a `downgrade` in the language of the following comment.

Reply:
Yes, you can pass the `is_superuser=false` to `downgrade` the user, Could you let me know if you see an error in my description or in the implementation, please.

Reply:
My observation related to your descriptive text. You describe this as general user maintenance, but really it's just about super-users.